k9mail-library/src/main/java/com/fsck/k9/mail/filter/EOLConvertingOutputStream.java

```diff
@@ -7,8 +7,10 @@
 public class EOLConvertingOutputStream extends FilterOutputStream {
     private static final int CR = '\r';
     private static final int LF = '\n';
-    private int lastChar;
-    private static final int IGNORE_LF = Integer.MIN_VALUE;
+
+
+    private int[] lastBytes = new int[2];
```

Comment:
I had to stop and ponder what the order of these bytes might be. Maybe just use `lastByte` and `secondLastByte` to be more clear, or add a comment about its order.